fcd/main.cpp

```diff
@@ -598,9 +499,10 @@ class Main {
         optimizeAndTransformPasses =
             interactivelyEditPassPipeline(editor, passNames);
       } else {
-        errs() << getProgramName() << ": environment has no EDITOR variable; "
-                                      "pass pipeline can't be edited "
-                                      "interactively\n";
+        errs() << getProgramName()
```

Comment:
Glog?

Reply:
Same as explicit `std::` and `llvm::`, will be a part of a separate PR.